### ISSUE TITLE
`getChallengeRange` can be a private function instead of a private static method.

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/query-string-machine#48]
For #84

In NumberPlayGameLevel.ts:

```ts
  private static getChallengeRange( levelNumber: number, levelChallengeRange: number ): Range {
    const minimumChallengeNumber = ( levelNumber - 1 ) * levelChallengeRange + 1;
    const maximumChallengeNumber = levelNumber * levelChallengeRange;
    return new Range( minimumChallengeNumber, maximumChallengeNumber );
  }
```

Since it's private, this does not need to be a static method.  Move it outside of the constructor to make it a private function.  In general, if you find yourself creating a `private static` method, it should probably be a private function.

**Comment by pixelzoom** [comment]
Same for:
* CountingGameLevel.getRandomPlayObjectType
* NumberPlayGameModel.getLevelNumbers
* the 8 `private static` methods in Subitizer

**Comment by samreid** [comment]
I was about to remove all usages of `private static` functions and add a lint rule to prevent more from being created, then I realized that our project has probably more than 100 of them. So I wanted to ask: should we just accept/embrace `private static` functions and close this issue?

**Comment by pixelzoom** [comment]
>  should we just accept/embrace `private static` methods and close this issue? 

Yes, I think that's fine.  I prefer to use a private `function` in these circumstances in JavaScript/TypeScript, but I don't have a problem with other devs using a `private static` method for the same purpose.  And in some other OO languages (e.g. Java) a `private static` method would be the only option.
